storageFormats/eventMetadata.h

```diff
@@ -40,6 +40,10 @@ namespace rpwa {
 		const std::vector<std::string>& productionKinematicsParticleNames() const { return _productionKinematicsParticleNames; }
 		const std::vector<std::string>& decayKinematicsParticleNames() const { return _decayKinematicsParticleNames; }
 		const std::vector<std::string>& additionalSavedVariableLables() const { return _additionalSavedVariableLabels; }
+		const std::map<std::string,double>& additionalSavedValues() const { return _additionalSavedValues; }
+		double getAdditionalSavedValue(const std::string& name) const { return _additionalSavedValues.at(name); }
+		void setAdditionalSavedValue(const std::string& name, const double value) { _additionalSavedValues[name] = value; }
+		bool hasAdditionalSavedValue(const std::string& name) const { return _additionalSavedValues.find(name) != _additionalSavedValues.end(); }
```

Comment:
This would be shorter, but I think the semantic for find is more clear, because here we want to find whether the map has that key, not count it. 
Performance wise it does not matter.